eyeshade/workers/reports.js

```diff
@@ -74,5 +77,14 @@ exports.initialize = async (debug, runtime) => {
 
   if ((typeof process.env.DYNO === 'undefined') || (process.env.DYNO === 'worker.1')) {
     setTimeout(() => { daily(debug, runtime) }, 5 * 1000)
+    updateBalancesOnInterval(runtime)
   }
 }
+
+async function updateBalancesOnInterval (runtime) {
+  await updateBalances(runtime)
+  const now = +(new Date())
```

Comment:
Any reason to do this over `(new Date()).getTime()`?

Reply:
habit i suppose. i used to write really tersely. they all do the same thing
```
(new Date()).valueOf()
```